nixos/modules/services/video/mediamtx.nix

```
@@ -66,6 +66,7 @@ in
         Group = "mediamtx";
         SupplementaryGroups = lib.mkIf cfg.allowVideoAccess "video";
         ExecStart = "${cfg.package}/bin/mediamtx /etc/mediamtx.yaml";
+        Restart = "always";
```

```suggestion
        Restart = "on-failure";
```
https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Restart=